travis/gen-index.sh

```diff
@@ -154,7 +154,7 @@ fi
 
 # add the history results
 temp_work_dir=`mktemp -d -u`
-remote_url=`git config remote.origin.url`
+remote_url=`git config remote.origin.url` || remote_url="https://github.com/${owner}/${name}"
```

Comment:
Can you change it to `https://github.com/${owner}/${name}.git`? (note the `.git` ending)

Reply:
sure, will do